application/fin/netsuite.go

```diff
@@ -19,22 +21,62 @@ type NetSuite struct {
 	Period             int
 	Year               int
 	JournalDescription string
-	Transactions       []Transaction
+	Transactions       TransactionBlocks
 }
 
-func (n *NetSuite) AppendToBatch(t Transaction) {
+func (n *NetSuite) AppendToBatch(block string, t Transaction) {
 	if t.Amount != 0 {
-		n.Transactions = append(n.Transactions, t)
+		n.Transactions[block] = append(n.Transactions[block], t)
 	}
 }
 
-func (n *NetSuite) BatchToCSV() []byte {
+func (n *NetSuite) ToCSV() []byte {
 	var buf bytes.Buffer
 	buf.Write([]byte(netSuiteHeader1))
 	buf.Write([]byte(netSuiteHeader2))
 	buf.Write(n.summaryRow())
-	for i := range n.Transactions {
-		buf.Write(n.transactionRow(i))
+	for _, transactions := range n.Transactions {
+		for _, transaction := range transactions {
+			buf.Write(n.transactionRow(transaction))
+		}
+	}
+
+	return buf.Bytes()
+}
+
+func (n *NetSuite) ToZip() []byte {
+	// Create a buffer to write our archive to.
+	buff := new(bytes.Buffer)
+
+	// Create a new zip archive.
+	w := zip.NewWriter(buff)
+
+	for block, transactions := range n.Transactions {
+		f, err := w.Create(block + ".csv")
```

Comment:
```suggestion
	for blockName, transactions := range n.Transactions {
		f, err := w.Create(blockName + ".csv")
```
And if my above comment is right, rename `transactions` to `block`.

Reply:
It's a block of transactions, so I'm fine with either way.